### ISSUE TITLE
Cura 4.10 Not Loading Project Files From Earlier Versions

### ISSUE THREAD

**Issue by rachael7** [Ultimaker/Cura#10226]
### Application Version

4.10.0

### Platform

Windows 10

### Printer

Ultimaker S5

### Reproduction steps

1. Save a .3mf project in Cura 4.9, with modifications to profiles.
2. Open the project in Cura 4.10.

The attached project file was saved in Cura 4.9 and contains many profile changes. Loading it into your system should cause Cura to ask you how to handle the profile changes. Instead it will simply insert the STL into your current project.

### Actual results

When loading project files saved from Cura 4.9 into Cura 4.10, it does not bring in the material profiles or setting changes from the saved file. It also does not clear the plate first, placing the STL from the saved project file on the current plate, along with any objects that were already there. It gives no warnings about profile changes (the popup where it asks you to keep the current profile or use the settings in the project file). It doesn't even change the currently selected material to the one in the saved project file. In short, project files from Cura 4.9 are treated like ordinary STL files by Cura 4.10.

### Expected results

I expected that saved project files would be loaded by Cura 4.10 the way they have been in previous versions. First should be a popup asking if you want to keep or discard the current profile changes. Then it should clear the plate, bring in the STL(s) from the saved project file, select the materials/nozzles that were saved with the project file, and use the saved profile settings.

### Checklist of files to include

- [X] Log file
- [X] Project file

### Additional information & file uploads

[cura.log](https://github.com/Ultimaker/Cura/files/6925464/cura.log)
[Sample Project File From 4.9.zip](https://github.com/Ultimaker/Cura/files/6925487/Sample.Project.File.From.4.9.zip)

**Comment by rachael7** [comment]
Well I’ll be darned. I don’t remember changing it intentionally. Heck, I don’t remember even going into the settings since I’ve upgraded; but it sure did get changed somehow. Set that back and it worked just fine. Thanks for your help!

**Comment by rachael7** [comment]
> 
> Perhaps you clicked the "remember my choice" and then pressed import models on the screen that asks what to do with a project file. I've seen it happen a few times before. I'm glad that that was the issue :)

Definitely not - I've had it set to always open as project, ever since I started using Cura. Been that way so long I'd forgotten it was even an option and I couldn't recall ever seeing the dialog before I tested the setting change at your suggestion. I suspect it was some glitch in the upgrade process, so I'm going to double-check all my other settings, in case it's a corrupted settings file. At any rate, problem solved, and quickly too. Thanks again, I am in  your debt!
